viper.go

```diff
@@ -880,7 +881,9 @@ func (v *Viper) find(lcaseKey string) interface{} {
 			return cast.ToBool(flag.ValueString())
 		case "stringSlice":
 			s := strings.TrimPrefix(flag.ValueString(), "[")
-			return strings.TrimSuffix(s, "]")
+			s = strings.TrimSuffix(s, "]")
+			res, _ := readAsCSV(s)
```

Comment:
Thinking out loud: maybe the error should not be silently ignored but shouted loudly as `panic`? Any thoughts?